controller/task_queue/handler/tokenization.py

```diff
@@ -19,10 +21,20 @@ def __start_task(task: Dict[str, Any]) -> bool:
         return False
     project_id = task["project_id"]
 
+    if task["task_info"]["scope"] == RecordTokenizationScope.ATTRIBUTE.value:
+        attribute_item = attribute_db_bo.get(
+            project_id, task["task_info"]["attribute_id"]
+        )
+        if attribute_item is None or (
+            attribute_item.state
+            not in [AttributeState.INITIAL.value, AttributeState.FAILED.value]
+        ):
```

Comment:
This condition leads to an error because the calculated attribute is set to state usable directly after the tokenization is requested. So at the time when the tokenization is executed, the attribute is already in state usable.
I suggest to only check if the `attribute_item` is `None` and omitting the check for the states.

Reply:
changed to not USEABLE